docs/tutorials/gromacs/spack-gromacs.yaml

```diff
@@ -88,12 +88,12 @@ deployment_groups:
         # fi
         # spack buildcache keys --install --trust
 
-        spack config --scope defaults add config:build_stage:/apps/spack/spack-stage
+        spack config --scope defaults add config:build_stage:/opt/apps/spack/spack-stage
         spack config --scope defaults add -f /tmp/projections-config.yaml
         spack config --scope site add -f /tmp/slurm-external-config.yaml
 
-        spack install gcc@9.3.0 target=x86_64
-        spack load gcc@9.3.0 target=x86_64
+        spack install gcc@9.3.0 %gcc@8.5.0 target=x86_64
+        spack load gcc@9.3.0 %gcc@8.5.0 target=x86_64
```

Comment:
where is the 8.5 coming from. Is it needed?

Reply:
Removed.